excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/endpoint/exception/SpreadsheetValidationErrorMessageCreatorTest.java

```diff
@@ -757,26 +758,26 @@ public void eventEntityInvalidUserRole_createErrorMessageCalled_customMessageRet
     }
 
     @Test
-    public void eventUserRoleEntityInvalidUserRole_createErrorMessageCalled_defaultMessageReturned() {
+    public void eventEntityInvalidUserRole_createErrorMessageCalled_defaultMessageReturned() {
```

Comment:
Rename test when validation error `EventEntityInvalidUserRoleValidationError` is renamed

Reply:
EventEntityInvalidUserRoleValidationError